### PR TITLE
Build full feature on docs.rs

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: rendy/Cargo.toml [hunk 1]
```diff
@@ -97,4 +97,7 @@ required-features = ["base", "wsi-winit"]
 
 [[example]]
 name = "source_shaders"
-required-features = ["base", "wsi-winit"]
\ No newline at end of file
+required-features = ["base", "wsi-winit"]
+
+[package.metadata.docs.rs]
+features = ["base", "mesh-obj", "serde-1", "texture-image", "texture-palette", "spirv-reflection", "wsi-winit"]
```
Comment (omni-viral) [inline rendy/Cargo.toml]: You intentionally ommited `spirv-compilation`?
Otherwise just specify `full`

Comment (torkleyy) [reply rendy/Cargo.toml]: Yes, because I think it would need additional dependencies, right?

Comment (torkleyy) [reply rendy/Cargo.toml]: ninja I believe

Comment (Frizi) [reply rendy/Cargo.toml]: Ninja is only required on windows afaik

Comment (torkleyy) [reply rendy/Cargo.toml]: Ah right, and docs.rs only builts for Linux by default!

Comment (torkleyy) [inline rendy/Cargo.toml]:
```suggestion
features = ["full"]
```

**Pull request merged**